java/pom.xml

```diff
@@ -796,7 +796,7 @@
       <dependency>
         <groupId>org.junit.jupiter</groupId>
         <artifactId>junit-jupiter-api</artifactId>
-        <version>5.7.0</version>
+        <version>5.7.2</version>
```

Comment:
It seems that we need `junit.version` to prevent two separate PRs for `junit-jupiter-api` and `junit-jupiter-params`.